tests/test.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+test('my first test', async ({ page }) => {
+  
+  test.skip(!!process.env.CI, 'Test file is skipped in CI due to Cloudflare protection.');
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
```

Comment:
Let's add baseURL - 'https://practicesoftwaretesting.com' to the playwright config. This way you won't have to write full link in the test
```suggestion
  await page.goto('/auth/login');
```